easybuild/easyconfigs/b/binutils/binutils-2.25-GCC-4.9.2-binutils-2.25.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'binutils'
+version = '2.25'
+
+homepage = 'http://directory.fsf.org/project/binutils/'
+description = "binutils: GNU binary utilities"
+
+toolchain = {'name': 'GCC', 'version': '4.9.2-binutils-%(version)s'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [GNU_SOURCE]
+
+builddependencies = [
+    ('zlib', '1.2.8'),
+    ('binutils', version, '', True)
+]
+
+# make sure that system libraries are also considered by ld
+configopts = '--with-sysroot=/'
```

Comment:
no, I mean, fix order of `configopts` vs `preconfigopts`, like in the other binutils easyconfig where we link zlib statically

There is no versionsuffix yet here (the toolchain has one, sure), but I do agree that adding one like `-static-zlib` would only make the module name a lot longer with minimal added value.

So, let's just simply consistently link zlib statically in binutils (2.25) easyconfigs?

It makes sense to me, to avoid fixing the zlib version pretty hard for the entire software stack in such a low-level tool...